packages/next-app/src/wagmi/client.ts

```diff
@@ -1,24 +1,31 @@
 import { getDefaultWallets } from "@rainbow-me/rainbowkit";
-import { goerli, polygon } from "wagmi/chains";
+import { goerli, mainnet, polygon } from "wagmi/chains";
 import { publicProvider } from "wagmi/providers/public";
 
 import { configureChains, createClient } from "#/wagmi";
 
-const { chains, provider, webSocketProvider } = configureChains(
-  [polygon, goerli],
-  [publicProvider()]
-);
+export function getWagmiClient(routerPath: string) {
+  const walletChains =
+    routerPath === "/internalmanager"
+      ? [mainnet, goerli, polygon]
+      : [polygon, goerli];
```

Comment:
🤔 we could leave mainnet in all paths, but would just need show a message saying it's unsupported yet.